cypress/integration/find.spec.js

```diff
@@ -123,23 +140,59 @@ describe('find* dom-testing-library commands', () => {
     cy.findByText('New Page Loaded').should('exist')
   })
 
+  it('findByText should set the Cypress element to the found element', () => {
+    // This test is a little strange since snapshots show what element
+    // is selected, but snapshots themselves don't give access to those
+    // elements. I had to make the implementation specific so that the `$el`
+    // is the `subject` when the log is added and the `$el` is the `value`
+    // when the log is changed. It would be better to extract the `$el` from
+    // each snapshot
+
+    cy.on('log:changed', (attrs, log) => {
+      if (log.get('name') === 'findByText') {
+        expect(log.get('$el')).to.have.text('Button Text 1')
+      }
+    })
+
+    cy.findByText('Button Text 1')
+  })
+
   it('findByText should error if no elements are found', () => {
     const regex = /Supercalifragilistic/
-    const errorMessage = `Timed out retrying: Expected to find element: 'findByText(${regex})', but never found it.`
+    const errorMessage = `Unable to find an element with the text: /Supercalifragilistic/`
+    cy.on('fail', err => {
+      expect(err.message).to.contain(errorMessage)
+    })
+
+    cy.findByText(regex, {timeout: 100}) // Every find query is implicitly a `.should('exist')
```

Comment:
I think we should explicitly use `.should('exist')`

`cy.verifyUpcomingAssertions` has a default of existence if no other assertion is found. That's why it isn't explicitly needed. I like to err on the side of being more explicit, but I left it as is from previous code. I'm happy to change it if others agree

Reply:
I agree with you.

In the jest world, I recommend people wrap `get*` queries with assertions of existence even though they throw:

```javascript
expect(getByText(/success/i)).toBeInTheDocument()
```